spec/services/fetch_link_card_service_spec.rb

```diff
@@ -148,6 +149,14 @@
       end
     end
 
+    context 'with a URL of a page in ISO-8859-1 encoding, that charlock_holmes cannot detect' do
+      let(:status) { Fabricate(:status, text: 'Check out http://example.com/low_confidence_latin1') }
+
+      it 'decodes the HTML' do
+        expect(status.preview_card.title).to eq("Tofu á l'orange")
```

Comment:
I am confused at this assertion, are we not asserting this is decoded correctly?

Reply:
Nevermind, github decided to decode it as something else than utf-8, but the file's content is correct utf-8.
![image](https://github.com/mastodon/mastodon/assets/384364/657d6f91-2ec9-48e2-bd98-1627c1ea4180)